### PR TITLE
LPS 199149 TECHNICAL TASK v0 remove manual env var injection review

### PR DESCRIPTION
**Pull request by gamerson**
- LPS-199149 liferay-clarity-etc-spring-boot: removes manual env var injection from build.gradle
- LPS-199149 liferay-jethr0-etc-spring-boot: removes manual env var injection from build.gradle
- LPS-199149 liferay-marketplace-etc-spring-boot: removes manual env var injection from build.gradle
- LPS-199149 liferay-partner-etc-cron: removes manual env var injection from build.gradle
- LPS-199149 liferay-ticket-etc-cron: removes manual env var injection from build.gradle
- LPS-199149 liferay-ticket-etc-spring-boot: removes manual env var injection from build.gradle
- LPS-199149 try to stabilize test with retry/wait

### CODE REVIEW

**Comment by gamerson** [comment]
ci:test:liferay-sample-workspace

**Comment by gamerson** [comment]
ci:test:liferay-sample-workspace

**Comment by drewbrokke** [comment]
ci:test:liferay-sample-workspace

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/ClientExtensionsSamples.testcase [hunk 1]
```diff
@@ -190,9 +190,22 @@ definition {
 
 			echo(${springBootLogFileContent});
 
+			var found = false;
```
Comment (drewbrokke) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/ClientExtensionsSamples.testcase:193]: I think this has to be a string `"false"`, then the later comparison will look like:
```
if (${found} != "true") {
}
```

**Review by drewbrokke — CHANGES_REQUESTED** [review]

**Comment by drewbrokke** [comment]
See https://github.com/drewbrokke/liferay-portal/pull/1561

**Pull request closed without merging**